src/components/modals/ScheduleFiltersDialog.tsx

```diff
@@ -49,6 +50,8 @@ export const LOCATIONS_COLOR = blue;
 
 export const CATEGORIES_COLOR = pink;
 
+export const CLASS_TIME_COLOR = purple;
```

Comment:
```suggestion
export const EXCLUDE_CLASS_TIME_COLOR = purple;
```

Reply:
:heavy_check_mark:

Reply:
Looks like the rename was not completed, hence the build error 🙃

Reply:
God damnit VSCode...